caas/kubernetes/provider/admissionregistration.go

```diff
@@ -135,74 +134,43 @@ func (k *kubernetesClient) EnsureMutatingWebhookConfiguration(cfg *admissionregi
 // nil.
 func (k *kubernetesClient) ensureMutatingWebhookConfigurationV1beta1(cfg *admissionregistrationv1beta1.MutatingWebhookConfiguration) (func(), error) {
 	cleanUp := func() {}
-	out, err := k.createMutatingWebhookConfiguration(cfg)
+	api := k.client().AdmissionregistrationV1beta1().MutatingWebhookConfigurations()
```

Comment:
This should just be v1 right?

Reply:
`ensureMutatingWebhookConfigurationV1beta1` is for handling `v1beta1` webhook spec provided in charm's podspec